integration-tests/load-tests/signatureRegistry.test.ts

```diff
@@ -0,0 +1,160 @@
+import "@frequency-chain/api-augment";
+import assert from "assert";
+import { createKeys, signPayloadSr25519, getBlockNumber, generateAddKeyPayload, createAndFundKeypair, devAccounts } from "../scaffolding/helpers";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { AddKeyData, ExtrinsicHelper } from "../scaffolding/extrinsicHelpers";
+import { EXISTENTIAL_DEPOSIT } from "../scaffolding/rootHooks";
+import { firstValueFrom } from "rxjs";
+import { u64, Option } from "@polkadot/types";
+
+interface GeneratedMsa {
+    id: u64;
+    nonce: number;
+    controlKey: KeyringPair;
+}
+
+describe("MSA Initializer Load Tests", function () {
+
+    beforeEach(async function () {
+        await createBlock();
```

Comment:
There are a lot of createBlocks in here to support the manual sealing.

This also makes a lot of the code complex as we have to create a promise (or lots of them) then await them to be resolved after we create a block.